pipelinerun-logs/config/deployment.yaml

```diff
@@ -30,3 +33,6 @@ spec:
         - "9999"
         ports:
         - containerPort: 9999
+      nodeSelector:
```

Comment:
Can we just enable this on all nodes?

Reply:
Yes it's possible.  Initially I didn't want to interfere with any existing workloads running on the non-workload-identity nodes so I created a separate node pool just for the feature.

I'm not clear on what the risks are for workloads on migrated nodes - i.e. what is the default behaviour if a service account wasn't originally included in a Deployment that is then migrated to a workload identity node?